pkgs/applications/misc/paperjam/default.nix

```
@@ -0,0 +1,41 @@
+{ lib , stdenv, fetchgit, qpdf, libpaper, asciidoc, }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "paperjam";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "git://git.ucw.cz/paperjam.git";
+    # by recording the exact commit, we can embed it into the binary
+    # this can't be done automatically as fetchgit removes .git directory
+    rev = "90173f436b39bfe39491aa356a218ac185618d1c";
```

Looking at the [Makefile](http://www.ucw.cz/gitweb/?p=paperjam.git;a=blob;f=Makefile;h=32c31b43ba2cb4caf057a197ab8e0884161049ea;hb=HEAD#l4), it the binary would contain a tag name when a commit with a tag is used. Might as well use

```suggestion
    rev = "v${finalAttrs.version}";
```

That will also make it easier to update the package with `passthru.updateScript = gitUpdater { };` or other similar tooling.